backend/publisher.py

```diff
@@ -118,8 +121,8 @@ def send_message(self,type_,damsg):
             }
         try:
             self.caller._myAppSession.publish('com.opentrons.robot_to_browser',json.dumps(msg))
-        except:
-            logging.error("error trying to send_message")
+        except Exception as e:
+            logging.exception("error trying to send_message")
```

Comment:
should us the `logger` variable retrieved from `getLogger()` at the top of this module

There are conflicts like this with how `logging` is being used in this PR vs master